docs/user-guide/rug/editors.md

```diff
@@ -126,45 +126,45 @@ public class ${this.controller_class_name}Controller {
-of the editor. The second argument of the `#!typescript @Editor` decorator is a 
-short description of the editor. The following line uses the 
-`#!typescript @Tags` decorator to apply some tags to our editor so people can 
-search for it more easily. Using the `#!typescript @Tags` decorator is optional 
+The first few lines group the Rug typings our script will be using throughout
+(lines 1&ndash;4). Next, we declare our editor through TypeScript [decorators][]
+(line 6). The first argument of the `#!typescript @Editor` decorator is the name
+of the editor. This the public visible and discoverable name of the Rug. This
+name, along with the editors group and repository, form the fully-qualified name
+of the editor. The second argument of the `#!typescript @Editor` decorator is a
+short description of the editor. The following line uses the
+`#!typescript @Tags` decorator to apply some tags to our editor so people can
+search for it more easily. Using the `#!typescript @Tags` decorator is optional
 but highly recommended.
 
 [decorators]: https://www.typescriptlang.org/docs/handbook/decorators.html
 
-We then define the class which implements our editor (line 8). An editor 
-implements the `#!typescript EditProject` interface. That interface requires the 
-`#!typescript edit(Project)` method to be defined (line 28). It is convention 
+We then define the class which implements our editor (line 8). An editor
+implements the `#!typescript EditProject` interface. That interface requires the
+`#!typescript edit(Project)` method to be defined (line 28). It is convention
 for the editor and the class that implements it to have the same name.
 
-You will likely want to customize your Rugs' input to tune their output 
-according to user-supplied values. Such customization is achieved through 
-parameters that your editor must declare in the class via the 
-`#!typescript @Parameter` decorator (lines 10, 16 and 22). This decorator 
-provides all the metadata of the parameter whereas the decorated variable 
-declaration provides its name and default value, if any. The 
-`#!typescript @Parameter` decorator accepts a single argument, a JavaScript 
-object. That object accepts [properties][paramprops] documented in the 
-[conventions][rugconv] but, only the `pattern` property is required. A pattern 
-is an [anchored][] regular expression that validates the input. Here we rely on 
+You will likely want to customize your Rugs' input to tune their output
+according to user-supplied values. Such customization is achieved through
+parameters that your editor must declare in the class via the
+`#!typescript @Parameter` decorator. This decorator
+provides all the metadata of the parameter whereas the decorated variable
+declaration provides its name and default value, if any. The
+`#!typescript @Parameter` decorator accepts a single argument, a JavaScript
+object. That object accepts [properties][paramprops] documented in the Rug
```

Comment:
replace `accepts` with `must implement the `Parameter` interface from `@atomist/rug/operations/RugOperations.ts`. These are not conventions and are enforced.